src/consumer.rs

```diff
@@ -1454,10 +1501,9 @@ impl<Exe: Executor> ConsumerBuilder<Exe> {
             consumer_name,
             batch_size,
             unacked_message_resend_delay,
-            namespace,
-            topic_refresh,
             consumer_options,
             dead_letter_policy,
+            ..
```

Comment:
if there are more fields in the builder, they should be used, not silently ignored

Reply:
done with `unused_variables` macro